### PR TITLE
Serialization Docs: Better intro to custom serializer section

### PR DESCRIPTION
**Pull request by sebastianharko**
The intro sentence to the Custom Serializer section need a bit rephrasing. 

The original is "So, lets say that you want to create your own Serializer, you saw the docs.serialization.MyOwnSerializer in the config example above?" .... can we come up with a better intro paragraph here ?

### CODE REVIEW

File: akka-docs/rst/scala/serialization.rst [hunk 1]
```diff
@@ -89,13 +89,12 @@ For more information, have a look at the ``ScalaDoc`` for ``akka.serialization._
 Customization
 =============
 
-So, lets say that you want to create your own ``Serializer``,
-you saw the ``docs.serialization.MyOwnSerializer`` in the config example above?
+The first code snippet on this page contains a configuration file that references a custom serializer ``docs.serialization.MyOwnSerializer``. How would we go about creating such a custom serializer ?
```
Comment (johanandren) [inline akka-docs/rst/scala/serialization.rst]: weird space before question mark

**Review by johanandren — APPROVED** [review]
LGTM with one tiny note

File: akka-docs/rst/scala/serialization.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/scala/serialization.rst]: I propose to just edit such things and merge ;-)

Comment (ktoso) [inline akka-docs/rst/scala/serialization.rst]: serialization_docs__better_intro_to_custom_serializer_section_by_sebastianharko_ _pull_request__22653_ _akka_akka

Comment (johanandren) [inline akka-docs/rst/scala/serialization.rst]: Ah, never noticed that little pen before, thanks

**Review by ktoso — APPROVED** [review]

**Pull request merged**